### DISCUSSION TITLE
All new additions to the public profile should be opt-in

### DISCUSSION THREAD

**Discussion by bwcorey** [community/community#27954] [Other Feature Feedback, Questions, & Ideas]
I don't want my public profile changing out from under me. Please make all future additions to the public profile page opt-in, not opt-out.

Edit: to clarify this is brought on by the achievements being added to the public profile by default.

**Comment by radar-blip** [comment]
Dont know if it will help but you do have these options here:

![privacy](https://user-images.githubusercontent.com/39196767/182292375-4bf4b61f-bf4d-49ce-a7f9-b094842f3fb5.png)
